packages/account/src/Components/file-uploader-container/file-uploader-container.tsx

```diff
@@ -11,32 +11,29 @@ type TFileUploaderContainer = {
     onError?: (error_message: string) => void;
 };
 
-const FileUploaderContainer = observer(
-    ({ examples, files_description, onFileDrop, onError }: TFileUploaderContainer) => {
-        const {
-            ui: { is_mobile },
-        } = useStore();
-        return (
-            <div className='file-uploader__container' data-testid='dt_file_uploader_container'>
-                {files_description}
-                {examples}
-                <Text size={is_mobile ? 'xxs' : 'xs'} as='div' className='file-uploader__file-title' weight='bold'>
-                    <Localize i18n_default_text='Upload file' />
+const FileUploaderContainer = ({ examples, files_description, onFileDrop, onError }: TFileUploaderContainer) => {
+    const { isMobile } = useDevice();
```

Comment:
will the design for tablet and desktop be same in this component?

Reply:
Hey @utkarsha-deriv
As we don't have design for all screens and I checked on tablet view xs is too small for tablet view.